ni_measurementlink_generator/pyproject.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 repository = "https://github.com/ni/measurementlink-python/"
 license = "MIT"
 classifiers = [
-    "Development Status :: 3 - Alpha",
```

Comment:
I think we should do a last prerelease before we update the development status.

Reply:
@bkeryan reset you for approval. I think now is the right time to make this change.